bot/cogs/random_cog/random_cog.py

```diff
@@ -171,5 +174,22 @@ async def raffle(self, ctx, time: typing.Optional[Duration] = 5, *, reason):
                     embed = discord.Embed(title = 'RAFFLE', color=Colors.ClemsonOrange, description = description)
                     await msg.edit(embed = embed)
 
+    @commands.command(aliases=['xkcd'])
+    async def relevant(self, ctx):
+        """
+        Generates a possibly relevant xkcd.
+        https://c.xkcd.com/random/comic/ is a random comic from the xkcd catalogue
+        """
+        async with aiohttp.ClientSession() as session:
+            async with await session.get(url="https://c.xkcd.com/random/comic/") as resp:
+                if(resp.status == 200):
+                    await ctx.send(resp.url)
```

Comment:
should the comic itself be deletable?

Reply:
The embeds are typically pretty small but yeah, I can make it deleteable